osm2streets/src/lib.rs

```diff
@@ -602,6 +587,16 @@ impl Road {
         let right = center.shift_from_center(total_width, total_width / 2.0)?;
         Ok((left, right))
     }
+
+    // TODO Don't take our own ID
```

Comment:
The other PR moving IDs into the struct would be helpful here and in a few other places!